features/step_definitions/buyer_steps.rb

```diff
@@ -277,3 +277,8 @@ def login_form
 When(/^as a developer$/) do
   step 'the current domain is foo.3scale.localhost'
 end
+
+When "the buyer wants to sign up" do
```

Comment:
Style: We should use single quotes when there's no string interpolation.

Reply:
This is correct, although it's my fault because I thought it was the standard for Cucumber and I've been doing it for ages 😬

Reply:
<img src="https://media4.giphy.com/media/yvBAuESRTsETqNFlEl/giphy.gif?cid=ecf05e477iz13hhuiu7e7sxbaod2mba0k1v13icxpxs394hh&rid=giphy.gif&ct=g" width="240" height="240"/>